.github/workflows/build.yml

```diff
@@ -62,6 +62,18 @@ jobs:
             type=semver,pattern={{major}}
             type=sha
 
+      - name: Prepare args
```

Comment:
Please test if the proposed changes to the 'Build and push' step work. If they do, we actually don't need this step at all and it can be safely removed